Bootstrap_task/styles/_nav.scss

```diff
@@ -0,0 +1,41 @@
+.navbar {
+  padding-top: 19px;
+  padding-bottom: 19px;
+  .navbar-brand {
```

Comment:
you could use "&" instead writing the whole class name:
`.navbar {
   ....

    &-brand {
        ....
    }
}`

Reply:
oh, i thought we can use & for class if its navbar__brand, not navbar-brand. Thanks